src/status_im/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -44,34 +43,48 @@
       :on-scroll-to-index-failed    identity
       :render-fn                    asset-component}]))
 
-(defn- tab-view
-  [search-text selected-tab]
-  (case selected-tab
-    :tab/assets       [asset-list search-text]
-    :tab/collectibles [quo/empty-state
-                       {:title           (i18n/label :t/no-collectibles)
-                        :description     (i18n/label :t/no-collectibles-description)
-                        :placeholder?    true
-                        :container-style (style/empty-container-style (safe-area/get-top))}]))
-
 (defn- search-input
+  [search-text on-change-text]
+  [rn/view {:style style/search-input-container}
+   [quo/input
+    {:small?         true
+     :placeholder    (i18n/label :t/search-assets)
+     :icon-name      :i/search
+     :value          search-text
+     :on-change-text on-change-text}]])
+
+(defn collectibles-grid
   [search-text]
-  (let [on-change-text #(reset! search-text %)]
-    (fn []
-      [rn/view {:style style/search-input-container}
-       [quo/input
-        {:small?         true
-         :placeholder    (i18n/label :t/search-assets)
-         :icon-name      :i/search
-         :value          @search-text
-         :on-change-text on-change-text}]])))
+  (let [collectibles (rf/sub [:wallet/current-viewing-account-collectibles-filtered search-text])]
+    [collectibles-tab/view
+     {:collectibles         collectibles
+      :filtered?            true
+      :on-collectible-press (fn [collectible-id]
+                              (js/alert (str "Collectible to send: \n"
+                                             collectible-id
+                                             "\nNavigation not implemented yet")))}]))
+
+(defn- tab-view
+  [search-text selected-tab on-change-text]
+  (let [unfiltered-collectibles (rf/sub [:wallet/current-viewing-account-collectibles])
+        show-search-input?      (or (= selected-tab :tab/assets)
+                                    (and (= selected-tab :tab/collectibles)
+                                         (seq unfiltered-collectibles)))]
+    [:<>
+     (when show-search-input?
+       [search-input search-text on-change-text])
```

Comment:
The search input is not always shown in the collectibles view, it's shown only when there are collectibles.
On the other hand, the assets tab is always showing the search input.